package.json

```diff
@@ -26,65 +28,62 @@
     "dist/*"
   ],
   "scripts": {
-    "build": "rollup -c ./rollup.config.js",
-    "test": "ts-mocha 'test/**/*.ts'",
+    "build": "rollup -c rollup.config.mjs",
+    "test": "ts-mocha -p ./tsconfig-test.json 'test/**/*.ts'",
     "lint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
     "fix": "eslint --fix 'src/**/*.ts' 'test/**/*.ts'",
-    "release:ci:prepatch": "release-it patch --preRelease=alpha --ci",
-    "release:ci:preminor": "release-it minor --preRelease=alpha --ci",
-    "release:ci:premajor": "release-it major --preRelease=alpha --ci",
-    "release:ci:bump-prerelease": "release-it --preRelease --ci",
+    "release:ci:prepatch": "release-it patch --preRelease=alpha-$(git rev-parse --short HEAD) --ci",
+    "release:ci:preminor": "release-it minor --preRelease=alpha-$(git rev-parse --short HEAD) --ci",
+    "release:ci:premajor": "release-it major --preRelease=alpha-$(git rev-parse --short HEAD) --ci",
     "release:ci:alpha->beta": "release-it --preRelease=beta --ci",
     "release:ci:beta->rc": "release-it --preRelease=rc --ci",
     "release:ci:patch": "release-it patch --ci",
     "release:ci:minor": "release-it minor --ci",
     "release:ci:major": "release-it major --ci"
   },
-  "dependencies": {
-    "js-cookie": "^3.0.1",
-    "live-connect-common": "^1.0.0"
+  "peerDependencies": {
```

Comment:
Why going with peer dependencies?

Reply:
They might be also set by `live-connect` or `live-connect-builder`.

Reply:
Nope. You are right @mschuwalow

Reply:
But I will investigate it more closely